omp/test/solver/lower_trs_kernels.cpp

```diff
@@ -132,8 +165,8 @@ TEST_F(LowerTrs, ApplyIsEquivalentToRef)
     initialize_data(59, 3);
```

Comment:
need blank line

Reply:
I think here it is according to the AAA pattern.

Reply:
you are right. it is an arrange part.